test/test_surface_waves.jl

```diff
@@ -3,7 +3,7 @@ function instantiate_surface_waves()
     ∂t_vˢ(z, t) = exp(z/20) * cos(t)
     ∂z_uˢ(z, t) = exp(z/20) * cos(t)
     ∂z_vˢ(z, t) = exp(z/20) * cos(t)
-    surface_waves = SurfaceWaves.UniformStokesDrift(∂t_uˢ=∂t_uˢ, ∂t_vˢ=∂t_vˢ,
+    stokes_drift = SurfaceWaves.UniformStokesDrift(∂t_uˢ=∂t_uˢ, ∂t_vˢ=∂t_vˢ,
                                                     ∂z_uˢ=∂z_uˢ, ∂z_vˢ=∂z_vˢ)
```

Comment:
```suggestion
                                                   ∂z_uˢ=∂z_uˢ, ∂z_vˢ=∂z_vˢ)
```

fixes the alignment issue.